rules/opentitan.bzl

```diff
@@ -564,7 +564,7 @@ def opentitan_rom_binary(
         targets.extend(opentitan_binary(
             name = devname,
             deps = deps + dev_deps,
-            extract_sw_logs_db = extract_sw_logs_db and device == "sim_dv",
+            extract_sw_logs_db = extract_sw_logs_db and device.startswith("sim_"),
```

Comment:
Why just `sim_` instead of `sim_db`?

Reply:
Currently, we only backdoor write LOG messages on DV simulations, bypassing the (slow) UART, however, in the future, it would be more performant if we did the same for Verilator simulations. While this would require updates to the Verilator testbench, it would also require the log database file to be present, so I altered this to generate this database text file for both "sim" targets (DV and Verilator).Additionally, by generating these for both sim targets, it simplified the logic in `opentitan_test.bzl` on lines 296--304. If you prefer, I can remove it and only make that change if/when we optimize Verilator backdoor message logging?